inputs/costs/investment_costs_households_storage_space_heating.ad

```diff
@@ -0,0 +1,19 @@
+- query =
+    UPDATE(
+      V(
+        households_space_heater_heatpump_air_water_electricity,
+        households_space_heater_heatpump_ground_water_electricity,
+        households_space_heater_hybrid_heatpump_air_water_electricity,
+        storage
+      ),
+      cost_per_mwh,
+      DIVIDE(USER_INPUT(), 0.001)
```

Comment:
Did you mean for this to be `DIVIDE(USER_INPUT(), 1000)` to convert from euro/kWh (what the user inputs) to euro/MWh (what ETEngine uses)?

Reply:
yes indeed. 1 euro/kWh = 1000 euro/mWh right?
So that is time 1000, or divide by 0.001.

Or should I not do this conversion here?

Reply:
> So that is times 1000, or divide by 0.001.

Sorry, you're completely right. What was I thinking... 🙄

It might be clearer to multiply (`USER_INPUT() * 1000` or `PRODUCT(USER_INPUT(), 1000`), but I don't have a strong opinion. It's fine as it is if you prefer using `DIVIDE` here.